### PR TITLE
Fixed a possible NRE in Hotkey.TryParse

### PR DESCRIPTION
**Pull request by Mailaender**
Discovered by Coverity.

### CODE REVIEW

File: OpenRA.Game/Input/Hotkey.cs [hunk 1]
```diff
@@ -24,6 +24,9 @@ public bool IsValid()
 		public static bool TryParse(string s, out Hotkey result)
 		{
 			result = Invalid;
+			if (string.IsNullOrWhiteSpace(s))
```
Comment (penev92) [inline OpenRA.Game/Input/Hotkey.cs]: Are you sure you don't want `NullOrEmpty` here?
What about pressing Spacebar?

Comment (Mailaender) [inline OpenRA.Game/Input/Hotkey.cs]: .Split is actually the offender below and it will also make ` ` an impossible parse.

**Comment by abcdefg30** [comment]
:+1:

File: OpenRA.Game/Input/Hotkey.cs (hunk 1 above)
Comment (pchote) [inline OpenRA.Game/Input/Hotkey.cs]: Spacebar corresponds to the string `SPACE`, so there's no problem here.

**Comment by pchote** [comment]
This won't fix any real crashes, but that doesn't mean we shouldn't fix this :+1:

**Comment by pchote** [comment]
https://github.com/OpenRA/OpenRA/wiki/Changelog-%28bleed%29/_compare/cc48afa23705add3002e89c6e9c34d7445f3070e%5E...cc48afa23705add3002e89c6e9c34d7445f3070e

**Pull request merged**